app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,306 @@
+    <string name="menu_share">Sdílet</string>
+    <string name="menu_delete">Odstranit</string>
+    <string name="menu_search_online">Hledat online</string>
+    <string name="menu_choose_lyrics">Vybrat jiné texty</string>
+
+    <!-- Dialog -->
+    <string name="dialog_title_details">Podrobnosti</string>
+    <string name="media_id">ID média</string>
+    <string name="mime_type">Typ MIME</string>
+    <string name="codecs">Kodeky</string>
+    <string name="bitrate">Přenosová rychlost</string>
+    <string name="sample_rate">Vzorkovací frekvence</string>
+    <string name="loudness">Hlučnost</string>
+    <string name="volume">Hlasitost</string>
+    <string name="file_size">Velikost souboru</string>
+    <string name="unknown">Neznámé</string>
+
+    <string name="dialog_title_edit_lyrics">Upravit texty</string>
+    <string name="dialog_title_search_lyrics">Hledat texty</string>
+    <string name="dialog_title_choose_lyrics">Vybrat texty</string>
+
+    <string name="dialog_title_edit_song">Upravit skladbu</string>
+    <string name="song_title">Název skladby</string>
+    <string name="song_artists">Umělci skladby</string>
+    <string name="error_song_title_empty">Název skladby nemůže být prázdný.</string>
+    <string name="error_song_artist_empty">Umělec skladby nemůže být prázdný.</string>
+    <string name="dialog_button_save">Uložit</string>
+
+    <string name="dialog_title_create_playlist">Vytvořit playlist</string>
+    <string name="text_view_hint_playlist_name">Název playlistu</string>
+    <string name="error_playlist_name_empty">Název playlistu nemůže být prázdný.</string>
+
+    <string name="dialog_title_edit_artist">Upravit umělce</string>
+    <string name="text_view_hint_artist_name">Jméno umělce</string>
+    <string name="error_artist_name_empty">Jméno umělce nemůže být prázdné.</string>
+
+    <string name="dialog_title_duplicate_artist">Duplicitní umělci</string>
+    <string name="dialog_msg_duplicate_artist">Umělec %1$s již existuje.</string>
+
+    <string name="dialog_title_choose_playlist">Vybrat playlist</string>
+
+    <string name="dialog_title_edit_playlist">Upravit playlist</string>
+
+    <string name="dialog_title_choose_backup_content">Vybrat obsah zálohy</string>
+    <string name="dialog_title_choose_restore_content">Vybrat obsah obnovení</string>
+    <string name="choice_preferences">Předvolby</string>
+    <string name="choice_database">Databáze</string>
+    <string name="choice_downloaded_songs">Stažené skladby</string>
+    <string name="message_backup_create_success">Záloha úspěšně vytvořena</string>
+    <string name="message_backup_create_failed">Nepodařilo se vytvořit zálohu</string>
+    <string name="message_restore_failed">Nepodařilo se obnovit zálohu</string>
+
+    <!-- Notification -->
+    <string name="channel_name_playback">Hudební přehrávač</string>
+    <string name="channel_name_download">Stáhnout</string>
+
+    <!-- Noun -->
+    <plurals name="song_count">
```

Comment:
the following quantities should also be defined: few

Reply:
Done

Reply:
The build says "the following quantity should also be defined: many".

Reply:
Okay, done, too.

Reply:
Oops, didn't see those as well, sorry, I'll check the whole file